gazelle/modules_mapping/test_generator.py

```diff
@@ -0,0 +1,54 @@
+import pathlib
+import unittest
+
+from generator import Generator
+
+
+class GeneratorTest(unittest.TestCase):
+    def test_generator(self):
+        whl = pathlib.Path(
+            pathlib.Path(__file__).parent, "testdata", "pytest-7.1.1-py3-none-any.whl"
+        )
+        gen = Generator(None, None, {}, False)
+        gen.dig_wheel(whl)
```

Comment:
Creating wheels using `rules_python` would be preferable to using upstream wheels. We have plenty of wheels available in `examples/wheels`.